crates/tiny/src/config.rs

```diff
@@ -64,6 +66,28 @@ pub(crate) struct Defaults {
     pub(crate) tls: bool,
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq)]
+#[serde(untagged)]
+pub(crate) enum Chan {
+    /// Channel specified by name only
+    Name(String),
+    /// Channel specified by name with extra configuration (used by tui crate)
+    WithExtra {
+        name: String,
+        #[serde(flatten)]
+        extra: HashMap<String, Value>,
```

Comment:
Why is this needed? I think TUI is parsing the config file again, it's not using this, no?

Reply:
To make the `tiny` `Config` generic and the `libtiny_tui` `Config` aware of the specific values.
My comment above:
> Can replace with libtiny_tui's `Chan` (requires exporting the type), if we want to export that type, or keep it like this.